src/components/consul/selectors/ImageSelector.js

```diff
@@ -14,19 +17,54 @@ import { WrapperRow } from '../../Wrapper';
 
 const { IMAGE_TYPE_REGEX } = consts;
 
-export const ImageSelector = ({ control, field, item }) => {
+const deleteImageAlert = (onDeleteImage) =>
+  Alert.alert(
+    texts.consul.startNew.deleteAttributesAlertTitle,
+    texts.consul.startNew.imageDeleteAlertBody,
+    [
+      {
+        text: texts.consul.abort,
+        onPress: () => null,
+        style: 'cancel'
+      },
+      {
+        text: texts.consul.startNew.deleteAttributesButtonText,
+        onPress: () => onDeleteImage(),
```

Comment:
if the param for `deleteImageAlert` would be `onPress` instead of `onDeleteImage` this could be refactored to the following, as we pass no further params to the callback method:

```js
text: texts.consul.startNew.deleteAttributesButtonText,
onPress,
```

Reply:
[b2165fe](https://github.com/ikuseiGmbH/smart-village-app-app/pull/540/commits/b2165fe121ef9f8a52809c0490e7fae9b0b7046a)